project/src/main.c

```diff
@@ -27,36 +33,46 @@ int main(int argc, const char** argv) {
     if (argc < 3) {
         return ERR_ARGS_COUNT;
     }
-
-    int Test_case = atoi(argv[1]);
+    char* parg;
+    int Test_case = strtol(argv[1], &parg, DECIMAL_NUMBER_SYSTEM);
     const char* data;
     data = argv[2];
```

Comment:
По возможности нужно совмещать определение и инициализацию.
```suggestion
    const char* data = argv[2];
```